package.json

```diff
@@ -12,7 +12,7 @@
     "dev": "npm run prebuild && ng serve --environment=dev --host 0.0.0.0",
     "stage": "npm run prebuild && ng serve --environment=stage --host 0.0.0.0",
     "prod": "npm run prebuild && ng serve --environment=prod --host 0.0.0.0",
-    "prebuild": "cp ./node_modules/vizabi/build/dist/vizabi.css ./src/assets/vizabi.css && cp ./node_modules/vizabi-barrankchart/build/dist/barrankchart.css ./src/assets/barrankchart.css",
+    "prebuild": "cp ./node_modules/vizabi/build/dist/vizabi.css ./src/assets/vizabi.css && cp ./node_modules/vizabi-barrankchart/build/dist/barrankchart.css ./src/assets/barrankchart.css && cp ./node_modules/vizabi-bubblechart/build/dist/bubblechart.css ./src/assets/bubblechart.css && cp ./node_modules/vizabi-mountainchart/build/dist/mountainchart.css ./src/assets/mountainchart.css && cp ./node_modules/vizabi-linechart/build/dist/linechart.css ./src/assets/linechart.css",
```

Comment:
@kuguarpwnz Maybe let's move CSS copying to the external bash script file? Cause line is growing and in a couple, extra tools - will be huge.